init.sql

```diff
@@ -1,79 +1,76 @@
-CREATE TABLE Account (
-	accountID varchar(20) SERIAL PRIMARY KEY, /* Serial does automatic increment? https://chartio.com/resources/tutorials/how-to-define-an-auto-increment-primary-key-in-postgresql/ */
-	password varchar(50) NOT NULL
-);
-
-CREATE TABLE User (
+CREATE TABLE AppUser (
 	username varchar(50) PRIMARY KEY,
+	password varchar(50) NOT NULL,
     firstName varchar(20) NOT NULL,
     lastName varchar(20) NOT NULL,
     address varchar(50) NOT NULL,
-    password varchar(50) NOT NULL,
-    phoneNumber integer NOT NULL
+    phoneNumber varchar(20) NOT NULL
 );
 
 CREATE TABLE Driver (
-	username varchar(50) PRIMARY KEY REFERENCES User ON DELETE cascade,
+	username varchar(50) PRIMARY KEY REFERENCES AppUser ON DELETE cascade,
 	d_rating integer,
 	license_no integer NOT NULL
 );
 
 CREATE TABLE Passenger (
-	username varchar(50) PRIMARY KEY REFERENCES User ON DELETE cascade,
+	username varchar(50) PRIMARY KEY REFERENCES AppUser ON DELETE cascade,
 	p_rating integer
 );
 
 CREATE TABLE Model (
-	name varchar(20) PRIMARY KEY,
-    size integer
+	name varchar(20),
+	brand varchar(50),
+    size INTEGER NOT NULL,
+	PRIMARY KEY (name, brand)
 );
 
 CREATE TABLE Car (
 	plateNumber varchar(20) PRIMARY KEY,
-    colours  varchar(20)
+    colours  varchar(20) NOT NULL
 );
```

Comment:
To change the ER diagram such that each driver can now own multiple cars @shawn-t